generative_ai/rag.py

```diff
@@ -35,7 +35,17 @@ def create_corpus(
     # Initialize Vertex AI API once per session
     vertexai.init(project=project_id, location="us-central1")
 
-    corpus = rag.create_corpus(display_name=display_name, description=description)
+    # Configure embedding model
+    embedding_model_config = rag.EmbeddingModelConfig(
+        publisher_model="publishers/google/models/textembedding-gecko@003"
```

Comment:
Can we update this to textembedding-004?

Reply:
done